readme.md

```diff
@@ -26,16 +26,20 @@ gulp.task('default', function () {
 
 ## Options
 
+Type: `object`
+
 ### mode
 
 Type: `string`
 
-Set mode on the extracted files, i.e `{mode: '755'}`.
+Set permissions on the extracted files, i.e `{mode: '755'}`.
 
 ### strip
 
 Type: `number`
 
+Removes, or flattens, a number of leading directories from each file in the archive.
```

Comment:
I am intending to provide alternative terminology by including "or flattens," not suggest that either is possible with gulp-decompress.

Perhaps you would be more comfortable with "Removes—or flattens—a number..." (the em dash) or "Removes (or flattens) a number...?"